cmake/RunTests.cmake

```diff
@@ -38,7 +38,7 @@ set(ENV{SYSTEM_NAME} ${SYSTEM_NAME})
 execute_process(
   COMMAND ${BUSTED_PRG} ${TEST_TAG} ${TEST_FILTER} -v -o ${BUSTED_OUTPUT_TYPE}
     --lua=${LUA_PRG} --lazy --helper=${TEST_DIR}/${TEST_TYPE}/preload.lua
-    --lpath=${BUILD_DIR}/?.lua ${TEST_PATH}
+    --lpath="${BUILD_DIR}/?.lua" --lpath="?.lua" ${TEST_PATH}
```

Comment:
I thought cmake magically takes care of the quotes, which is why that isn't common. Did you find different?

Reply:
mb I was careless. I removed the quotes.